velox/connectors/hive/iceberg/tests/IcebergReadTest.cpp

```diff
@@ -35,15 +35,27 @@ namespace facebook::velox::connector::hive::iceberg {
 class HiveIcebergTest : public HiveConnectorTestBase {
  public:
   void assertPositionalDeletes(
-      const std::vector<int64_t>& deleteRows,
+      const std::vector<std::vector<int64_t>>& deleteRowVectors,
       bool multipleBaseFiles = false) {
+    std::vector<int64_t> deleteRows;
+    uint32_t numDeleteRows = 0;
+    for (const std::vector<int64_t>& deleteRowVector : deleteRowVectors) {
+      numDeleteRows += deleteRowVector.size();
+    }
+
+    deleteRows.reserve(numDeleteRows);
```

Comment:
If `numDeleteRows` is computed only to use here inside `reserve`, we can avoid the above loop. insert below will reallocate if needed.

Reply:
Yes, I will remove the same, only added it for performance reasons but since it's a test we can probably ignore that.